Sports.py

```diff
@@ -66,81 +265,195 @@ def blackRectangle(self, offscreen_canvas, xcord, ycord, rect_width, rect_height
             for y in range(ycord, ycord + rect_height):
                 offscreen_canvas.SetPixel(x, y, 0, 0, 0)  # Drawing black rectangle
                 
-    def grayRectangle(self, offscreen_canvas, xcord, ycord, rect_width, rect_height):
-        for x in range(xcord, xcord + rect_width):
-            for y in range(ycord, ycord + rect_height):
-                offscreen_canvas.SetPixel(x, y, 1, 50, 1)  # Drawing black rectangle
 
     def clearScreen(self,offscreen_canvas):
         offscreen_canvas.Clear()
         
-    def displayPlayerStats(self, offscreen_canvas):
         
+    def determinePercentColor(number):
+        
+        if number < 25:
+             return graphics.Color(255,0,0) #RED
+        elif number < 50:
+            return graphics.Color(255,165,0) #ORANGE
+        elif number <75:
+            return graphics.Color(255,255,0) #YELLLO
+        else:
+            return graphics.Color(0,128,0) #GREEN

```

Comment:
is this method used? seems to not be implemented in other main functions.

Reply:
sounds good! not used will delete!